source/Sale.h

```diff
@@ -55,11 +65,11 @@ void Sale<Item>::Load(const DataNode &node, const Set<Item> &items)
 		else if(add && child.Size() == 2 && child.Token(1) != "description")
 			this->insert(items.Get(child.Token(1)));
 		else if(add && child.Size() >= 3 && child.Token(1) == "description")
-			description += child.Token(2) + '\n';
+			this->AppendDescription(child.Token(2) + '\n');
 		else if(remove && child.Size() >= 2 && child.Token(1) == "description")
-			description = "";
+			this->SetDescription("");
 		else if(token == "description" && child.Size() >= 2)
-			description += child.Token(1) + '\n';
+			this->AppendDescription(child.Token(1) + '\n');
```

Comment:
you dont need an appenddescription function, because description is a private member of sale already, and it wouldnt make sense for this to be possible from somewhere else

only the getdescription was useful and needed when making description private

Reply:
I read the above as a request for getters and setters for more encapsulation. I don't care either way they have been removed.

Reply:
sorry, that is my fault

Reply:
NP. Have worked with people that take encapsulation very seriously before so thought nothing of it.